client/signup/jetpack-connect/controller.js

```diff
@@ -12,6 +12,7 @@ import i18n from 'i18n-calypso';
  * Internal Dependencies
  */
 import JetpackConnect from './index';
+import JetpackNewSite from './new-site/index';
```

Comment:
Are the `/index` necessary? I think we could remove them and it would still work the same way.